messages/sfdxurl.store.md

```diff
@@ -18,10 +18,16 @@ NOTE: The "<%= config.bin %> org display --verbose" command displays the refresh
 
 You can also create a JSON file that has a top-level property named sfdxAuthUrl whose value is the authorization URL. Finally, you can create a normal text file that includes just the URL and nothing else.
 
+Alternatively, you can pipe the SFDX authorization URL through standard input by using the --sfdx-url-stdin flag and providing the '-' character as the value.
+
 # flags.sfdx-url-file.summary
 
 Path to a file that contains the Salesforce DX authorization URL.
 
+# flags.sfdx-url-stdin.summary
+
+Provide '-' as the value to pipe the Salesforce DX authorization URL through standard input (stdin).
```

Comment:
```suggestion
Specify '-' as the value to pipe the Salesforce DX authorization URL through standard input (stdin).
```

Reply:
I'm not thrilled with this summary in general, but if that's how this flag works, then I guess this is how the summary will be too.